util/util_test.go

```diff
@@ -68,6 +68,27 @@ func TestRemoveSortedDuplicates(t *testing.T) {
 	assert.True(t, sort.IsSorted(SortableInt64(normalTest)))
 }
 
+func TestCountAndRemoveSortedDuplicates(t *testing.T) {
```

Comment:
If the above suggestion is implemented we can remove `TestRemoveSortedDuplicates`